src/ray/object_manager/plasma/store.cc

```diff
@@ -274,7 +274,7 @@ uint8_t *PlasmaStore::AllocateMemory(size_t size, MEMFD_TYPE *fd, int64_t *map_s
     pointer = reinterpret_cast<uint8_t *>(
         PlasmaAllocator::DiskMemalignUnlimited(kBlockSize, size));
     if (pointer == nullptr) {
-      RAY_LOG(FATAL) << "Plasma fallback allocator failed, likely out of disk space.";
+      RAY_LOG(ERROR) << "Plasma fallback allocator failed, likely out of disk space.";
```

Comment:
Why did we change this to ERROR? The machine might be already hanging or in a weird state if the disk is out of space.

Reply:
I recently fixed this to propagate a normal PlasmaStoreFull error to the user. They will just get that instead of a segfault.